src/app/shared/components/btn-groups/btn-groups.component.html

```diff
@@ -1,24 +1,24 @@
 <div class="btn-groups">
   <button
-    routerLink="/kiev/details"
+    routerLink="/{{weatherInfo?.city}}/details"
```

Comment:
ok we don't need refactoring.
just rename btn-groups to forecast-btn-group.
And one more thing here we need use only city